x11-packages/resynthesizer/build.sh

```diff
@@ -0,0 +1,31 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/bootchk/resynthesizer
+TERMUX_PKG_DESCRIPTION="Suite of gimp plugins for texture synthesis"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=3.0.0
+TERMUX_PKG_SRCURL=git+https://github.com/bootchk/resynthesizer.git
+TERMUX_PKG_GIT_BRANCH=resynthesizer3
+TERMUX_PKG_DEPENDS="gimp, python"
+TERMUX_PKG_BUILD_DEPENDS="xorgproto"
+TERMUX_PKG_BUILD_IN_SRC=true
```

Comment:
We can probably also enable auto-updates while we're here.
```suggestion
TERMUX_PKG_AUTO_UPDATE=true
TERMUX_PKG_BUILD_IN_SRC=true
```